eng/pipelines/test-windows-job-single-machine.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       displayName: 'Install .NET 9 Runtime'
       inputs:
         packageType: runtime
-        version: 9.0.0-preview.3.24172.9
+        version: 9.0.0-rc.2.24473.5
```

Comment:
Man I wish it were more straightforward to flow the sdk version from global.json here. This is obviously prone to fall out of date, but, I'm not sure we can justify the investigation effort at this point either.

Reply:
There is a useGlobalJson task parameter but surely no use if global.json is not on disk on the test machine.